2025-09-11 22:32:06,266 - config.database - INFO - Connected to database: verification_system
2025-09-11 22:32:06,727 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-11 22:32:06,728 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-11 22:32:06,748 - werkzeug - INFO -  * Restarting with stat
2025-09-11 22:32:14,098 - config.database - INFO - Connected to database: verification_system
2025-09-11 22:32:14,476 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 22:32:14,886 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 22:32:57,555 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/product_routes.py', reloading
2025-09-11 22:33:00,537 - werkzeug - INFO -  * Restarting with stat
2025-09-11 22:33:11,679 - config.database - INFO - Connected to database: verification_system
2025-09-11 22:33:12,208 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 22:33:12,212 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 22:33:25,988 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/__init__.py', reloading
2025-09-11 22:33:28,191 - werkzeug - INFO -  * Restarting with stat
2025-09-11 22:33:36,604 - config.database - INFO - Connected to database: verification_system
2025-09-11 22:33:36,987 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 22:33:36,990 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 22:38:32,551 - config.database - INFO - Connected to database: verification_system
2025-09-11 22:38:33,476 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-11 22:38:33,477 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-11 22:38:33,482 - werkzeug - INFO -  * Restarting with stat
2025-09-11 22:38:46,039 - config.database - INFO - Connected to database: verification_system
2025-09-11 22:38:46,726 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 22:38:46,732 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:04:16,209 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:04:16,574 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-11 23:04:16,575 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-11 23:04:16,581 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:04:24,925 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:04:25,281 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:04:25,283 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:10:42,526 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/product_service.py', reloading
2025-09-11 23:10:45,883 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:11:09,493 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:11:10,087 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:11:10,110 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:11:12,637 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/__init__.py', reloading
2025-09-11 23:11:15,894 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:11:29,485 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:11:31,959 - services.product_service - INFO - Database indexes created successfully
2025-09-11 23:11:31,959 - services.product_service - INFO - ProductService initialized successfully
2025-09-11 23:11:32,129 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:11:32,134 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:11:38,578 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/__init__.py', reloading
2025-09-11 23:11:41,059 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:11:51,495 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:11:55,978 - services.product_service - INFO - Database indexes created successfully
2025-09-11 23:11:55,980 - services.product_service - INFO - ProductService initialized successfully
2025-09-11 23:11:56,316 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:11:56,323 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:15:49,872 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/route_registry.py', reloading
2025-09-11 23:15:51,864 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:15:58,761 - config.database - ERROR - Database setup error: All nameservers failed to answer the query _mongodb._tcp.ecommercecluster.aoxl38d.mongodb.net. IN SRV: Server Do53:127.0.0.53@53 answered REFUSED
2025-09-11 23:21:34,152 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:21:36,409 - services.product_service - INFO - Database indexes created successfully
2025-09-11 23:21:36,415 - services.product_service - INFO - ProductService initialized successfully
2025-09-11 23:21:36,620 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-11 23:21:36,622 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-11 23:21:36,635 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:21:45,004 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:21:48,080 - services.product_service - INFO - Database indexes created successfully
2025-09-11 23:21:48,080 - services.product_service - INFO - ProductService initialized successfully
2025-09-11 23:21:48,206 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:21:48,210 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:31:47,786 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-11 23:31:50,518 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:32:01,199 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:32:04,600 - services.product_service - INFO - Database indexes created successfully
2025-09-11 23:32:04,604 - services.product_service - INFO - ProductService initialized successfully
2025-09-11 23:32:04,727 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:32:04,728 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:32:09,378 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-11 23:32:12,379 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:32:22,839 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:32:25,085 - services.product_service - INFO - Database indexes created successfully
2025-09-11 23:32:25,085 - services.product_service - INFO - ProductService initialized successfully
2025-09-11 23:32:25,246 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:32:25,249 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:32:31,247 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-11 23:32:33,627 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:32:46,256 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:32:49,391 - services.product_service - INFO - Database indexes created successfully
2025-09-11 23:32:49,395 - services.product_service - INFO - ProductService initialized successfully
2025-09-11 23:32:49,917 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:32:49,928 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:51:56,367 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-11 23:51:58,394 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:52:07,459 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:52:10,337 - services.product_service - INFO - Database indexes created successfully
2025-09-11 23:52:10,337 - services.product_service - INFO - ProductService initialized successfully
2025-09-11 23:52:10,460 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:52:10,463 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-11 23:52:57,346 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-11 23:53:00,051 - werkzeug - INFO -  * Restarting with stat
2025-09-11 23:53:12,206 - config.database - INFO - Connected to database: verification_system
2025-09-11 23:53:14,740 - services.product_service - INFO - Database indexes created successfully
2025-09-11 23:53:14,741 - services.product_service - INFO - ProductService initialized successfully
2025-09-11 23:53:14,902 - werkzeug - WARNING -  * Debugger is active!
2025-09-11 23:53:14,905 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 00:08:26,460 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/utils/validators.py', reloading
2025-09-12 00:08:28,621 - werkzeug - INFO -  * Restarting with stat
2025-09-12 00:08:40,576 - config.database - INFO - Connected to database: verification_system
2025-09-12 00:08:43,649 - services.product_service - INFO - Database indexes created successfully
2025-09-12 00:08:43,653 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 00:08:43,932 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 00:08:43,937 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 00:15:35,899 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/utils/validators.py', reloading
2025-09-12 00:15:40,092 - werkzeug - INFO -  * Restarting with stat
2025-09-12 00:15:51,065 - config.database - INFO - Connected to database: verification_system
2025-09-12 00:15:54,119 - services.product_service - INFO - Database indexes created successfully
2025-09-12 00:15:54,120 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 00:15:54,229 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 00:15:54,231 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 00:49:20,681 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/integration_routes.py', reloading
2025-09-12 00:49:24,242 - werkzeug - INFO -  * Restarting with stat
2025-09-12 00:49:54,929 - config.database - INFO - Connected to database: verification_system
2025-09-12 00:49:58,410 - services.product_service - INFO - Database indexes created successfully
2025-09-12 00:49:58,410 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 00:49:58,536 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 00:49:58,538 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 00:50:52,310 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/integration_routes.py', reloading
2025-09-12 00:50:54,026 - werkzeug - INFO -  * Restarting with stat
2025-09-12 00:51:09,018 - config.database - INFO - Connected to database: verification_system
2025-09-12 00:51:11,876 - services.product_service - INFO - Database indexes created successfully
2025-09-12 00:51:11,877 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 00:51:12,003 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 00:51:12,005 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 00:51:53,759 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/integration_routes.py', reloading
2025-09-12 00:51:57,498 - werkzeug - INFO -  * Restarting with stat
2025-09-12 00:52:16,205 - config.database - INFO - Connected to database: verification_system
2025-09-12 00:52:18,805 - services.product_service - INFO - Database indexes created successfully
2025-09-12 00:52:18,806 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 00:52:19,078 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 00:52:19,087 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 00:52:49,514 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/integration_routes.py', reloading
2025-09-12 00:52:51,433 - werkzeug - INFO -  * Restarting with stat
2025-09-12 00:53:00,870 - config.database - INFO - Connected to database: verification_system
2025-09-12 00:53:03,161 - services.product_service - INFO - Database indexes created successfully
2025-09-12 00:53:03,170 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 00:53:03,525 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 00:53:03,529 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 00:53:05,748 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/integration_routes.py', reloading
2025-09-12 00:53:09,549 - werkzeug - INFO -  * Restarting with stat
2025-09-12 00:53:30,163 - config.database - INFO - Connected to database: verification_system
2025-09-12 00:53:34,062 - services.product_service - INFO - Database indexes created successfully
2025-09-12 00:53:34,066 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 00:53:34,448 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 00:53:34,456 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 00:53:37,010 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/integration_routes.py', reloading
2025-09-12 00:53:40,904 - werkzeug - INFO -  * Restarting with stat
2025-09-12 00:53:58,051 - config.database - INFO - Connected to database: verification_system
2025-09-12 00:54:01,578 - services.product_service - INFO - Database indexes created successfully
2025-09-12 00:54:01,583 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 00:54:01,813 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 00:54:01,817 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 00:54:02,970 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/integration_routes.py', reloading
2025-09-12 00:54:05,267 - werkzeug - INFO -  * Restarting with stat
2025-09-12 00:54:29,412 - config.database - INFO - Connected to database: verification_system
2025-09-12 00:54:47,497 - services.product_service - INFO - Database indexes created successfully
2025-09-12 00:54:47,499 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 00:54:47,603 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 00:54:47,604 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 01:00:51,145 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/app.py', reloading
2025-09-12 01:00:52,505 - werkzeug - INFO -  * Restarting with stat
2025-09-12 01:39:06,604 - config.database - INFO - Connected to database: verification_system
2025-09-12 01:39:08,333 - services.product_service - INFO - Database indexes created successfully
2025-09-12 01:39:08,334 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 01:45:22,884 - config.database - INFO - Connected to database: verification_system
2025-09-12 01:45:25,441 - services.product_service - INFO - Database indexes created successfully
2025-09-12 01:45:25,441 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 01:53:13,139 - config.database - INFO - Connected to database: verification_system
2025-09-12 01:53:15,897 - services.product_service - INFO - Database indexes created successfully
2025-09-12 01:53:15,898 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 01:53:16,162 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-12 01:53:16,163 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-12 01:53:16,167 - werkzeug - INFO -  * Restarting with stat
2025-09-12 01:53:23,082 - config.database - INFO - Connected to database: verification_system
2025-09-12 01:53:25,240 - services.product_service - INFO - Database indexes created successfully
2025-09-12 01:53:25,241 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 01:53:25,357 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 01:53:25,358 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 01:57:19,617 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-12 01:57:22,909 - werkzeug - INFO -  * Restarting with stat
2025-09-12 01:57:36,954 - config.database - INFO - Connected to database: verification_system
2025-09-12 01:57:39,264 - services.product_service - INFO - Database indexes created successfully
2025-09-12 01:57:39,266 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 01:57:39,453 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 01:57:39,457 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 02:27:42,781 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/app.py', reloading
2025-09-12 02:27:46,174 - werkzeug - INFO -  * Restarting with stat
2025-09-12 02:27:59,221 - config.database - INFO - Connected to database: verification_system
2025-09-12 02:28:03,807 - services.product_service - INFO - Database indexes created successfully
2025-09-12 02:28:03,808 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 02:28:03,934 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 02:28:03,935 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 02:28:06,127 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/app.py', reloading
2025-09-12 02:28:08,716 - werkzeug - INFO -  * Restarting with stat
2025-09-12 02:28:18,742 - config.database - INFO - Connected to database: verification_system
2025-09-12 02:28:23,215 - services.product_service - INFO - Database indexes created successfully
2025-09-12 02:28:23,216 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 15:39:43,710 - config.database - INFO - Connected to database: verification_system
2025-09-12 15:39:46,992 - services.product_service - INFO - Database indexes created successfully
2025-09-12 15:39:46,992 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 15:39:47,373 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-12 15:39:47,374 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-12 15:39:47,378 - werkzeug - INFO -  * Restarting with stat
2025-09-12 15:40:01,052 - config.database - INFO - Connected to database: verification_system
2025-09-12 15:40:03,666 - services.product_service - INFO - Database indexes created successfully
2025-09-12 15:40:03,666 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 15:40:03,883 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 15:40:04,731 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 15:50:51,377 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 15:50:51] "[31m[1mGET /auth/login HTTP/1.1[0m" 405 -
2025-09-12 15:50:54,864 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 15:50:54] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
2025-09-12 15:51:06,609 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 15:51:06] "[33mGET / HTTP/1.1[0m" 404 -
2025-09-12 15:53:34,036 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 15:53:34] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2025-09-12 15:57:51,714 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-12 15:57:55,894 - werkzeug - INFO -  * Restarting with stat
2025-09-12 15:58:11,415 - config.database - INFO - Connected to database: verification_system
2025-09-12 15:58:13,883 - services.product_service - INFO - Database indexes created successfully
2025-09-12 15:58:13,887 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 15:58:14,405 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 15:58:14,419 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 15:58:19,214 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-12 15:58:23,009 - werkzeug - INFO -  * Restarting with stat
2025-09-12 15:58:34,669 - config.database - INFO - Connected to database: verification_system
2025-09-12 15:58:37,296 - services.product_service - INFO - Database indexes created successfully
2025-09-12 15:58:37,296 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 15:58:37,528 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 15:58:37,532 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 15:58:42,409 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-12 15:58:45,335 - werkzeug - INFO -  * Restarting with stat
2025-09-12 15:58:54,449 - config.database - INFO - Connected to database: verification_system
2025-09-12 15:58:56,170 - services.product_service - INFO - Database indexes created successfully
2025-09-12 15:58:56,171 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 15:58:56,334 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 15:58:56,337 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 16:01:49,713 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 16:01:49] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2025-09-12 16:14:32,300 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/auth_service.py', reloading
2025-09-12 16:14:35,874 - werkzeug - INFO -  * Restarting with stat
2025-09-12 16:14:46,918 - config.database - INFO - Connected to database: verification_system
2025-09-12 16:14:49,810 - services.product_service - INFO - Database indexes created successfully
2025-09-12 16:14:49,811 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 16:14:49,957 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 16:14:49,959 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 16:14:50,246 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 16:14:50] "[31m[1mPOST /auth/login HTTP/1.1[0m" 401 -
2025-09-12 16:15:46,550 - config.database - INFO - Connected to database: verification_system
2025-09-12 16:16:02,710 - services.product_service - INFO - Database indexes created successfully
2025-09-12 16:16:02,710 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 16:16:03,057 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-12 16:16:03,060 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-12 16:16:03,071 - werkzeug - INFO -  * Restarting with stat
2025-09-12 16:16:11,478 - config.database - INFO - Connected to database: verification_system
2025-09-12 16:16:13,492 - services.product_service - INFO - Database indexes created successfully
2025-09-12 16:16:13,493 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 16:16:13,786 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 16:16:13,790 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 16:16:28,151 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 16:16:28] "[31m[1mPOST /auth/login HTTP/1.1[0m" 401 -
2025-09-12 16:19:19,888 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/auth_service.py', reloading
2025-09-12 16:19:24,268 - werkzeug - INFO -  * Restarting with stat
2025-09-12 16:19:55,782 - config.database - INFO - Connected to database: verification_system
2025-09-12 16:19:57,860 - services.product_service - INFO - Database indexes created successfully
2025-09-12 16:19:57,864 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 16:19:58,177 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 16:19:58,185 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 16:20:48,590 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/auth_service.py', reloading
2025-09-12 16:20:52,493 - werkzeug - INFO -  * Restarting with stat
2025-09-12 16:21:08,539 - config.database - INFO - Connected to database: verification_system
2025-09-12 16:21:11,377 - services.product_service - INFO - Database indexes created successfully
2025-09-12 16:21:11,378 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 16:21:11,613 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 16:21:11,620 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 16:23:00,747 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/auth_service.py', reloading
2025-09-12 16:23:03,667 - werkzeug - INFO -  * Restarting with stat
2025-09-12 16:36:22,576 - config.database - INFO - Connected to database: verification_system
2025-09-12 16:36:25,704 - services.product_service - INFO - Database indexes created successfully
2025-09-12 16:36:25,704 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 16:36:26,209 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-12 16:36:26,210 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-12 16:36:26,216 - werkzeug - INFO -  * Restarting with stat
2025-09-12 16:36:37,057 - config.database - INFO - Connected to database: verification_system
2025-09-12 16:36:39,263 - services.product_service - INFO - Database indexes created successfully
2025-09-12 16:36:39,264 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 16:36:39,724 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 16:36:39,727 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 16:37:05,217 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/utils/helper_functions.py', reloading
2025-09-12 16:37:10,305 - werkzeug - INFO -  * Restarting with stat
2025-09-12 16:37:31,166 - config.database - INFO - Connected to database: verification_system
2025-09-12 16:37:33,312 - services.product_service - INFO - Database indexes created successfully
2025-09-12 16:37:33,314 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 16:37:33,801 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 16:37:33,826 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 16:38:14,444 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 16:38:14] "[31m[1mPOST /auth/login HTTP/1.1[0m" 401 -
2025-09-12 16:45:18,267 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 16:45:18] "[31m[1mPOST /auth/login HTTP/1.1[0m" 401 -
2025-09-12 17:20:34,619 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/app.py', reloading
2025-09-12 17:20:37,123 - werkzeug - INFO -  * Restarting with stat
2025-09-12 17:20:59,146 - config.database - INFO - Connected to database: verification_system
2025-09-12 17:21:01,954 - services.product_service - INFO - Database indexes created successfully
2025-09-12 17:21:01,955 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 17:21:02,138 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-12 17:21:02,139 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-12 17:21:02,141 - werkzeug - INFO -  * Restarting with stat
2025-09-12 17:21:08,600 - config.database - INFO - Connected to database: verification_system
2025-09-12 17:21:11,497 - services.product_service - INFO - Database indexes created successfully
2025-09-12 17:21:11,498 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 17:21:11,652 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 17:21:11,678 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 17:23:17,480 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/app.py', reloading
2025-09-12 17:23:19,822 - werkzeug - INFO -  * Restarting with stat
2025-09-12 17:23:28,134 - config.database - INFO - Connected to database: verification_system
2025-09-12 17:23:31,346 - services.product_service - INFO - Database indexes created successfully
2025-09-12 17:23:31,353 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 17:23:31,729 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 17:23:31,740 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 17:23:52,412 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 17:23:52] "POST /auth/login HTTP/1.1" 200 -
2025-09-12 17:30:13,713 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/middleware/logging_middleware.py', reloading
2025-09-12 17:30:16,573 - werkzeug - INFO -  * Restarting with stat
2025-09-12 17:30:34,958 - config.database - INFO - Connected to database: verification_system
2025-09-12 17:30:36,909 - services.product_service - INFO - Database indexes created successfully
2025-09-12 17:30:36,910 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 17:30:37,082 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 17:30:37,085 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 17:31:42,429 - werkzeug - INFO - 127.0.0.1 - - [12/Sep/2025 17:31:42] "POST /auth/login HTTP/1.1" 200 -
2025-09-12 17:33:29,956 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/utils/helper_functions.py', reloading
2025-09-12 17:33:33,575 - werkzeug - INFO -  * Restarting with stat
2025-09-12 17:33:44,885 - config.database - INFO - Connected to database: verification_system
2025-09-12 17:33:46,905 - services.product_service - INFO - Database indexes created successfully
2025-09-12 17:33:46,905 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 17:33:47,145 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 17:33:47,147 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 17:34:09,437 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/utils/helper_functions.py', reloading
2025-09-12 17:34:11,443 - werkzeug - INFO -  * Restarting with stat
2025-09-12 17:34:28,513 - config.database - INFO - Connected to database: verification_system
2025-09-12 17:34:30,121 - services.product_service - INFO - Database indexes created successfully
2025-09-12 17:34:30,122 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 17:34:30,408 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 17:34:30,411 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 17:55:26,088 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/app.py', reloading
2025-09-12 17:55:30,540 - werkzeug - INFO -  * Restarting with stat
2025-09-12 17:56:04,763 - config.database - INFO - Connected to database: verification_system
2025-09-12 17:56:07,476 - services.product_service - INFO - Database indexes created successfully
2025-09-12 17:56:07,476 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 17:56:07,767 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 17:56:07,775 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 17:56:22,989 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-12 17:56:25,110 - werkzeug - INFO -  * Restarting with stat
2025-09-12 18:05:10,548 - config.database - INFO - Connected to database: verification_system
2025-09-12 18:05:12,275 - services.product_service - INFO - Database indexes created successfully
2025-09-12 18:05:12,275 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 18:05:12,846 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-12 18:05:12,847 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-12 18:05:12,879 - werkzeug - INFO -  * Restarting with stat
2025-09-12 18:05:24,275 - config.database - INFO - Connected to database: verification_system
2025-09-12 18:05:26,248 - services.product_service - INFO - Database indexes created successfully
2025-09-12 18:05:26,255 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 18:05:27,156 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 18:05:27,193 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 18:05:33,865 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-12 18:05:36,916 - werkzeug - INFO -  * Restarting with stat
2025-09-12 18:05:49,503 - config.database - INFO - Connected to database: verification_system
2025-09-12 18:05:52,486 - services.product_service - INFO - Database indexes created successfully
2025-09-12 18:05:52,490 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 18:05:53,399 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 18:05:53,420 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 18:05:57,663 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-12 18:06:00,808 - werkzeug - INFO -  * Restarting with stat
2025-09-12 18:06:14,902 - config.database - INFO - Connected to database: verification_system
2025-09-12 18:06:16,818 - services.product_service - INFO - Database indexes created successfully
2025-09-12 18:06:16,819 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 18:06:17,481 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 18:06:17,489 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 18:06:34,834 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-12 18:06:39,897 - werkzeug - INFO -  * Restarting with stat
2025-09-12 18:06:52,279 - config.database - INFO - Connected to database: verification_system
2025-09-12 18:06:54,939 - services.product_service - INFO - Database indexes created successfully
2025-09-12 18:06:54,940 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 18:06:55,419 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 18:06:55,422 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-12 18:07:12,568 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-12 18:07:15,417 - werkzeug - INFO -  * Restarting with stat
2025-09-12 18:07:27,185 - config.database - INFO - Connected to database: verification_system
2025-09-12 18:07:30,547 - services.product_service - INFO - Database indexes created successfully
2025-09-12 18:07:30,557 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 18:07:51,410 - config.database - INFO - Connected to database: verification_system
2025-09-12 18:07:53,628 - services.product_service - INFO - Database indexes created successfully
2025-09-12 18:07:53,629 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 18:07:53,893 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-12 18:07:53,894 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-12 18:07:53,897 - werkzeug - INFO -  * Restarting with stat
2025-09-12 18:08:01,474 - config.database - INFO - Connected to database: verification_system
2025-09-12 18:08:03,267 - services.product_service - INFO - Database indexes created successfully
2025-09-12 18:08:03,268 - services.product_service - INFO - ProductService initialized successfully
2025-09-12 18:08:03,605 - werkzeug - WARNING -  * Debugger is active!
2025-09-12 18:08:03,610 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:01:28,616 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:01:30,774 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:01:30,775 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:04:34,834 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:04:36,809 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:04:36,810 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:04:37,216 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 00:04:37,217 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 00:04:37,221 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:04:45,617 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:04:47,816 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:04:47,817 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:04:48,134 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:04:48,578 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:06:51,137 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-13 00:06:53,591 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:07:03,633 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:07:05,710 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:07:05,711 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:09:37,264 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:09:39,208 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:09:39,209 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:09:39,582 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 00:09:39,582 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 00:09:39,587 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:09:46,984 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:09:48,904 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:09:48,904 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:09:49,222 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:09:49,224 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:21:29,422 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:21:31,525 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:21:31,526 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:21:31,833 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 00:21:31,834 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 00:21:31,839 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:21:38,975 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:21:40,960 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:21:40,961 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:21:41,276 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:21:41,279 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:24:46,148 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:24:48,093 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:24:48,094 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:24:48,639 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 00:24:48,640 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 00:24:48,643 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:24:55,808 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:24:57,851 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:24:57,852 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:24:58,192 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:24:58,195 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:28:31,098 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-13 00:28:32,779 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:28:42,458 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:28:44,574 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:28:44,575 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:28:44,870 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:28:44,874 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:29:14,693 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:29:16,557 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:29:16,558 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:29:16,875 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 00:29:16,877 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 00:29:16,882 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:29:24,766 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:29:26,666 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:29:26,667 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:29:27,067 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:29:27,069 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:32:58,731 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-13 00:33:01,257 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:33:12,664 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:33:14,626 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:33:14,627 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:33:14,981 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:33:14,984 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:34:32,454 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/manufacturer_routes.py', reloading
2025-09-13 00:34:35,772 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:34:50,348 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:34:52,230 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:34:52,231 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:34:52,783 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:34:52,789 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:39:18,176 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-13 00:39:20,757 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:39:29,132 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:39:31,284 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:39:31,284 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:39:31,627 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:39:31,646 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:43:38,500 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-13 00:43:39,963 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:43:48,823 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:43:50,933 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:43:50,935 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:44:55,478 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:44:57,374 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:44:57,375 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:46:45,014 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:46:46,958 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:46:46,959 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:46:47,314 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 00:46:47,315 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 00:46:47,322 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:46:54,328 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:46:56,146 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:46:56,147 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:46:56,435 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:46:56,439 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 00:48:15,786 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/route_registry.py', reloading
2025-09-13 00:48:18,052 - werkzeug - INFO -  * Restarting with stat
2025-09-13 00:48:26,754 - config.database - INFO - Connected to database: verification_system
2025-09-13 00:48:28,727 - services.product_service - INFO - Database indexes created successfully
2025-09-13 00:48:28,728 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 00:48:29,055 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 00:48:29,060 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 01:19:52,678 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/auth_routes.py', reloading
2025-09-13 01:19:56,875 - werkzeug - INFO -  * Restarting with stat
2025-09-13 01:20:47,564 - config.database - INFO - Connected to database: verification_system
2025-09-13 01:20:49,536 - services.product_service - INFO - Database indexes created successfully
2025-09-13 01:20:49,537 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 01:20:49,944 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 01:20:49,944 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 01:20:49,947 - werkzeug - INFO -  * Restarting with stat
2025-09-13 01:20:57,261 - config.database - INFO - Connected to database: verification_system
2025-09-13 01:20:59,149 - services.product_service - INFO - Database indexes created successfully
2025-09-13 01:20:59,151 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 01:20:59,449 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 01:20:59,471 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 01:33:22,675 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/middleware/auth_middleware.py', reloading
2025-09-13 01:33:24,428 - werkzeug - INFO -  * Restarting with stat
2025-09-13 01:33:35,933 - config.database - INFO - Connected to database: verification_system
2025-09-13 01:33:37,825 - services.product_service - INFO - Database indexes created successfully
2025-09-13 01:33:37,828 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 01:33:38,509 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 01:33:38,512 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 01:33:46,299 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/middleware/auth_middleware.py', reloading
2025-09-13 01:33:48,884 - werkzeug - INFO -  * Restarting with stat
2025-09-13 01:33:59,823 - config.database - INFO - Connected to database: verification_system
2025-09-13 01:34:01,821 - services.product_service - INFO - Database indexes created successfully
2025-09-13 01:34:01,823 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 01:34:02,205 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 01:34:02,207 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 01:34:28,879 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/utils/formatters.py', reloading
2025-09-13 01:34:31,604 - werkzeug - INFO -  * Restarting with stat
2025-09-13 01:34:40,714 - config.database - INFO - Connected to database: verification_system
2025-09-13 01:34:42,734 - services.product_service - INFO - Database indexes created successfully
2025-09-13 01:34:42,735 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 01:56:39,784 - config.database - INFO - Connected to database: verification_system
2025-09-13 01:56:42,609 - services.product_service - INFO - Database indexes created successfully
2025-09-13 01:56:42,610 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 02:03:10,289 - config.database - INFO - Connected to database: verification_system
2025-09-13 02:03:12,513 - services.product_service - INFO - Database indexes created successfully
2025-09-13 02:03:12,513 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 02:11:22,290 - config.database - INFO - Connected to database: verification_system
2025-09-13 02:11:24,786 - services.product_service - INFO - Database indexes created successfully
2025-09-13 02:11:24,787 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 02:11:25,170 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 02:11:25,171 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 02:11:25,175 - werkzeug - INFO -  * Restarting with stat
2025-09-13 02:11:33,463 - config.database - INFO - Connected to database: verification_system
2025-09-13 02:11:36,740 - services.product_service - INFO - Database indexes created successfully
2025-09-13 02:11:36,742 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 02:11:37,057 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 02:11:37,059 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 02:16:40,678 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 02:16:40] "[35m[1mPOST /auth/login HTTP/1.1[0m" 500 -
2025-09-13 02:24:07,881 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/middleware/auth_middleware.py', reloading
2025-09-13 02:24:10,923 - werkzeug - INFO -  * Restarting with stat
2025-09-13 02:24:22,568 - config.database - INFO - Connected to database: verification_system
2025-09-13 02:24:24,651 - services.product_service - INFO - Database indexes created successfully
2025-09-13 02:24:24,652 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 02:24:24,893 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 02:24:24,895 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 02:25:03,331 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/middleware/auth_middleware.py', reloading
2025-09-13 02:25:06,089 - werkzeug - INFO -  * Restarting with stat
2025-09-13 02:25:16,123 - config.database - INFO - Connected to database: verification_system
2025-09-13 02:25:18,340 - services.product_service - INFO - Database indexes created successfully
2025-09-13 02:25:18,341 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 02:25:18,587 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 02:25:18,588 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 02:25:35,686 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/middleware/auth_middleware.py', reloading
2025-09-13 02:25:38,385 - werkzeug - INFO -  * Restarting with stat
2025-09-13 02:25:50,572 - config.database - INFO - Connected to database: verification_system
2025-09-13 02:25:52,774 - services.product_service - INFO - Database indexes created successfully
2025-09-13 02:25:52,775 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 02:25:53,032 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 02:25:53,034 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 02:25:55,967 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 02:25:55] "POST /auth/login HTTP/1.1" 200 -
2025-09-13 02:30:19,317 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 02:30:19] "POST /auth/login HTTP/1.1" 200 -
2025-09-13 02:42:28,814 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/integration_routes.py', reloading
2025-09-13 02:42:31,286 - werkzeug - INFO -  * Restarting with stat
2025-09-13 02:42:40,211 - config.database - INFO - Connected to database: verification_system
2025-09-13 02:42:42,128 - services.product_service - INFO - Database indexes created successfully
2025-09-13 02:42:42,128 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 02:42:42,415 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 02:42:42,417 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 02:46:45,511 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/middleware/auth_middleware.py', reloading
2025-09-13 02:46:47,241 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:18:09,314 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:18:15,431 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:18:15,432 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:18:15,772 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 03:18:15,773 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 03:18:15,780 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:18:27,237 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:18:29,524 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:18:29,528 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:18:30,080 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 03:18:30,082 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 03:18:32,734 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 03:18:32] "POST /auth/login HTTP/1.1" 200 -
2025-09-13 03:19:51,945 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 03:19:51] "[31m[1mGET /auth/manufacturer/profile HTTP/1.1[0m" 401 -
2025-09-13 03:20:46,534 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 03:20:46] "[31m[1mGET /auth/manufacturer/profile HTTP/1.1[0m" 401 -
2025-09-13 03:22:26,818 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 03:22:26] "[31m[1mGET /auth/manufacturer/profile HTTP/1.1[0m" 401 -
2025-09-13 03:28:47,690 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/auth_service.py', reloading
2025-09-13 03:28:49,877 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:29:02,847 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:29:06,584 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:29:06,585 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:29:06,898 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 03:29:06,902 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 03:29:20,877 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/auth_service.py', reloading
2025-09-13 03:29:24,307 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:29:36,419 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:29:38,481 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:29:38,482 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:29:38,791 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 03:29:38,794 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 03:29:43,113 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/auth_service.py', reloading
2025-09-13 03:29:45,982 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:29:55,350 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:29:57,575 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:29:57,576 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:29:58,003 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 03:29:58,006 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 03:30:31,386 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/auth_service.py', reloading
2025-09-13 03:30:34,272 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:30:44,743 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:31:11,931 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:31:14,212 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:31:14,212 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:31:14,691 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-13 03:31:14,694 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-13 03:31:14,711 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:31:22,691 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:31:24,895 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:31:24,896 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:31:25,286 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 03:31:25,290 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 03:31:40,702 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 03:31:40] "[31m[1mGET /auth/manufacturer/profile HTTP/1.1[0m" 401 -
2025-09-13 03:32:25,086 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 03:32:25] "[31m[1mGET /auth/manufacturer/profile HTTP/1.1[0m" 401 -
2025-09-13 03:33:48,898 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 03:33:48] "[31m[1mPOST /auth/login HTTP/1.1[0m" 401 -
2025-09-13 03:36:00,410 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/services/auth_service.py', reloading
2025-09-13 03:36:03,037 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:36:14,170 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:36:16,589 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:36:16,593 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:36:16,937 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 03:36:16,945 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 03:36:26,096 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 03:36:26] "POST /auth/login HTTP/1.1" 200 -
2025-09-13 03:37:09,509 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 03:37:09] "GET /auth/manufacturer/profile HTTP/1.1" 200 -
2025-09-13 03:46:20,545 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/manufacturer_routes.py', reloading
2025-09-13 03:46:25,058 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:46:36,735 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:46:38,744 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:46:38,745 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:46:39,062 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 03:46:39,065 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 03:59:08,638 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/product_routes.py', reloading
2025-09-13 03:59:10,650 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:59:20,939 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:59:23,366 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:59:23,367 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:59:23,851 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 03:59:23,862 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 03:59:31,528 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/product_routes.py', reloading
2025-09-13 03:59:34,007 - werkzeug - INFO -  * Restarting with stat
2025-09-13 03:59:41,570 - config.database - INFO - Connected to database: verification_system
2025-09-13 03:59:43,616 - services.product_service - INFO - Database indexes created successfully
2025-09-13 03:59:43,617 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 03:59:44,061 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 03:59:44,063 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 04:00:17,733 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/routes/manufacturer_routes.py', reloading
2025-09-13 04:00:20,516 - werkzeug - INFO -  * Restarting with stat
2025-09-13 04:00:32,119 - config.database - INFO - Connected to database: verification_system
2025-09-13 04:00:34,695 - services.product_service - INFO - Database indexes created successfully
2025-09-13 04:00:34,696 - services.product_service - INFO - ProductService initialized successfully
2025-09-13 04:00:35,122 - werkzeug - WARNING -  * Debugger is active!
2025-09-13 04:00:35,126 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-13 04:04:51,692 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 04:04:51] "GET /manufacturer/products HTTP/1.1" 200 -
2025-09-13 04:05:18,239 - werkzeug - INFO - 127.0.0.1 - - [13/Sep/2025 04:05:18] "GET /manufacturer/dashboard-stats HTTP/1.1" 200 -
2025-09-17 06:34:21,260 - config.database - INFO - Connected to database: verification_system
2025-09-17 06:34:23,157 - services.product_service - INFO - Database indexes created successfully
2025-09-17 06:34:23,158 - services.product_service - INFO - ProductService initialized successfully
2025-09-17 06:35:43,316 - config.database - INFO - Connected to database: verification_system
2025-09-17 06:36:05,930 - services.product_service - INFO - Database indexes created successfully
2025-09-17 06:36:05,933 - services.product_service - INFO - ProductService initialized successfully
2025-09-17 06:36:06,877 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.43.26:5000
2025-09-17 06:36:06,878 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2025-09-17 06:36:06,885 - werkzeug - INFO -  * Restarting with stat
2025-09-17 06:36:19,344 - config.database - INFO - Connected to database: verification_system
2025-09-17 06:36:22,159 - services.product_service - INFO - Database indexes created successfully
2025-09-17 06:36:22,161 - services.product_service - INFO - ProductService initialized successfully
2025-09-17 06:36:22,714 - werkzeug - WARNING -  * Debugger is active!
2025-09-17 06:36:23,535 - werkzeug - INFO -  * Debugger PIN: 143-327-685
2025-09-17 06:40:49,620 - werkzeug - INFO -  * Detected change in '/home/damilola/Desktop/product-verification-backend/config/blockchain.py', reloading
2025-09-17 06:40:51,471 - werkzeug - INFO -  * Restarting with stat
2025-09-19 15:59:31,344 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758293971, 4), 'signature': {'hash': b'\xceA\xe9\x87\xb0\xe3\x87\xd4\xf8\x90zwn\xbej\xf5\xe1\xcc\xb2\r', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758293971, 4)}
2025-09-19 15:59:31,348 - app - INFO - Database connection established
2025-09-19 15:59:31,349 - app - INFO - Services initialized
2025-09-19 15:59:40,759 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758293980, 10), 'signature': {'hash': b'\xdc\x95\x9d?\xec\xc1\x0b\xb5\x19&\xc9Fh\xd7t9_\xcc4\xe8', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758293980, 10)}
2025-09-19 15:59:40,761 - app - INFO - Database connection established
2025-09-19 15:59:40,762 - app - INFO - Services initialized
2025-09-19 15:59:41,242 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:01:46,692 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294106, 11), 'signature': {'hash': b'\xd0P,D\x1f\x930\xff\x9e\xb0\x9e\x08I=\x83jc7\xc7{', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294106, 11)}
2025-09-19 16:01:46,696 - app - INFO - Database connection established
2025-09-19 16:01:46,699 - app - INFO - Services initialized
2025-09-19 16:02:52,386 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294172, 3), 'signature': {'hash': b'\xa34\xf4c)\xcb\x8a\xf7\xce\xfaL\xa3\xa9m\xcej\xfbg\x10Y', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294172, 3)}
2025-09-19 16:02:52,387 - app - INFO - Database connection established
2025-09-19 16:02:52,389 - app - INFO - Services initialized
2025-09-19 16:03:05,304 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294185, 1), 'signature': {'hash': b'\xa5\x8d\xa8\x06\x1b\xa2q\xba\xec\x07\x13\x95\x08\xc4\xe8\xdbT\xb0M\xaf', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294185, 1)}
2025-09-19 16:03:05,306 - app - INFO - Database connection established
2025-09-19 16:03:05,306 - app - INFO - Services initialized
2025-09-19 16:03:05,922 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:09:31,607 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294571, 8), 'signature': {'hash': b'\xab\xb9\xf90\xf3\xcc\xc7b\xff\xb1\xb0\x9cCD\xdb\xdb\x1b\x03\xda\x8c', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294571, 8)}
2025-09-19 16:09:31,652 - app - INFO - Database connection established
2025-09-19 16:09:31,652 - app - INFO - Services initialized
2025-09-19 16:09:32,112 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:10:44,176 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294643, 11), 'signature': {'hash': b'\x95.\xfc\xf4\xa6\x00\xa9P\n\xbe\xe3\x1b\xe0\x95\x123G"\xe9G', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294643, 11)}
2025-09-19 16:10:44,182 - app - INFO - Database connection established
2025-09-19 16:10:44,182 - app - INFO - Services initialized
2025-09-19 16:10:44,727 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:10:59,814 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294659, 6), 'signature': {'hash': b'\x98\xc8\x98=\x85\x96&\x81\xba\xf0\xe0AR)\xafHW\xfdqc', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294659, 6)}
2025-09-19 16:10:59,816 - app - INFO - Database connection established
2025-09-19 16:10:59,817 - app - INFO - Services initialized
2025-09-19 16:11:00,126 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:11:23,982 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294683, 8), 'signature': {'hash': b'\xc7\x96\xcf\x81\x84f3\xbe\xc4\xbcmJ\xf4a\xc4\xff\x82B\xb9]', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294683, 8)}
2025-09-19 16:11:23,984 - app - INFO - Database connection established
2025-09-19 16:11:23,985 - app - INFO - Services initialized
2025-09-19 16:11:24,312 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:11:46,337 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294705, 11), 'signature': {'hash': b'\xb9\xdeR\xd3\xa7Vxl\x1d\xab\xe8^\xdc~\xf6\xd3T\x8bk\x9e', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294705, 11)}
2025-09-19 16:11:46,341 - app - INFO - Database connection established
2025-09-19 16:11:46,342 - app - INFO - Services initialized
2025-09-19 16:11:46,748 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:12:02,588 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294722, 3), 'signature': {'hash': b'c\xc7\xed\xf4\xb3\x86\x17j\x0f\xd3\xc4\x0e\xce?~\xcdul\x88R', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294722, 3)}
2025-09-19 16:12:02,595 - app - INFO - Database connection established
2025-09-19 16:12:02,600 - app - INFO - Services initialized
2025-09-19 16:12:03,202 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:12:17,809 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294737, 6), 'signature': {'hash': b'\xfd\x8c5_\x95\x86\x91\x17\xdf@B\xf3\x94=\x9a\xb4\x01\xd6C\xa9', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294737, 6)}
2025-09-19 16:12:17,810 - app - INFO - Database connection established
2025-09-19 16:12:17,811 - app - INFO - Services initialized
2025-09-19 16:12:18,091 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:12:41,703 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758294760, 12), 'signature': {'hash': b'\xeam\xc1\x02\xb4\tn\x03\xa8\xadm\xb0\xc5\xa2\xfe\xd4,\x03\xc6\xf8', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758294760, 12)}
2025-09-19 16:12:41,707 - app - INFO - Database connection established
2025-09-19 16:12:41,708 - app - INFO - Services initialized
2025-09-19 16:12:42,016 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:21:10,063 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758295269, 6), 'signature': {'hash': b"\x97\xaa\x9b\xeeMO\x9e@<\xf8\x8f\xe6\xafc'Qz{\x83N", 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758295269, 6)}
2025-09-19 16:21:10,070 - app - INFO - Database connection established
2025-09-19 16:21:10,075 - app - INFO - Services initialized
2025-09-19 16:21:10,531 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:21:35,698 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758295295, 8), 'signature': {'hash': b"\xda[^\xe1*\xe5\xb5\xc9\xab\xa0:\xdc\xdd\xaa\x92Y\xf1'\xdf\x13", 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758295295, 8)}
2025-09-19 16:21:35,704 - app - INFO - Database connection established
2025-09-19 16:21:35,708 - app - INFO - Services initialized
2025-09-19 16:21:36,280 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:21:46,505 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758295306, 9), 'signature': {'hash': b'\xd1\x18L\x83?\x1c/\xd5\xff\xbc3\xb9\x87\x16\xd1o\xd1\x1e\xc0\xed', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758295306, 9)}
2025-09-19 16:21:46,506 - app - INFO - Database connection established
2025-09-19 16:21:46,506 - app - INFO - Services initialized
2025-09-19 16:21:55,984 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1", sparse: true }, existing index: { v: 2, unique: true, key: { primary_email: 1 }, name: "primary_email_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758295315, 5), 'signature': {'hash': b"He\x0eiq5q'\xbcLM\xe2:\xbb'\x86\xd1\xd9\x9dR", 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758295315, 5)}
2025-09-19 16:21:55,986 - app - INFO - Database connection established
2025-09-19 16:21:55,986 - app - INFO - Services initialized
2025-09-19 16:21:56,356 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:28:31,889 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { serial_number: 1 }, name: "serial_number_1" }, existing index: { v: 2, key: { serial_number: 1 }, name: "serial_number_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { serial_number: 1 }, name: "serial_number_1" }, existing index: { v: 2, key: { serial_number: 1 }, name: "serial_number_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758295711, 31), 'signature': {'hash': b'?\xe8H5\x01\xd2/-\xef\xc0\xd1g\x86\x14\xd8\xbf\xc4\x04\xb4\xfb', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758295711, 31)}
2025-09-19 16:28:31,890 - app - INFO - Database connection established
2025-09-19 16:28:31,891 - app - INFO - Services initialized
2025-09-19 16:28:41,188 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { serial_number: 1 }, name: "serial_number_1" }, existing index: { v: 2, key: { serial_number: 1 }, name: "serial_number_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { serial_number: 1 }, name: "serial_number_1" }, existing index: { v: 2, key: { serial_number: 1 }, name: "serial_number_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758295720, 19), 'signature': {'hash': b'\x96B\x12\xaagt\xd4c\xf7\x84\x18\x1a\x87=?\xe6j@\xb7\x9f', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758295720, 19)}
2025-09-19 16:28:41,191 - app - INFO - Database connection established
2025-09-19 16:28:41,191 - app - INFO - Services initialized
2025-09-19 16:28:41,676 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:29:53,670 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { serial_number: 1 }, name: "serial_number_1" }, existing index: { v: 2, key: { serial_number: 1 }, name: "serial_number_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { serial_number: 1 }, name: "serial_number_1" }, existing index: { v: 2, key: { serial_number: 1 }, name: "serial_number_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758295793, 18), 'signature': {'hash': b'\x82]\x02k\xb9-V\xa3\xe7\x8d3_\xfe>A\xbc\xfe\\\xdc(', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758295793, 18)}
2025-09-19 16:29:53,671 - app - INFO - Database connection established
2025-09-19 16:29:53,676 - app - INFO - Services initialized
2025-09-19 16:30:06,054 - utils.database - ERROR - Error initializing database indexes: An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { serial_number: 1 }, name: "serial_number_1" }, existing index: { v: 2, key: { serial_number: 1 }, name: "serial_number_1" }, full error: {'ok': 0.0, 'errmsg': 'An existing index has the same name as the requested index. When index names are not specified, they are auto generated and can cause conflicts. Please refer to our documentation. Requested index: { v: 2, unique: true, key: { serial_number: 1 }, name: "serial_number_1" }, existing index: { v: 2, key: { serial_number: 1 }, name: "serial_number_1" }', 'code': 86, 'codeName': 'IndexKeySpecsConflict', '$clusterTime': {'clusterTime': Timestamp(1758295805, 36), 'signature': {'hash': b'\x88zv\xa5(\x16 \xa0\xcc\x96\x07\xc3\xc7U7\xfd}6\x94\xa0', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758295805, 36)}
2025-09-19 16:30:06,071 - app - INFO - Database connection established
2025-09-19 16:30:06,072 - app - INFO - Services initialized
2025-09-19 16:30:06,712 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:38:23,516 - utils.database - ERROR - Error initializing database indexes: Index build failed: e7d3444b-a548-46f0-b8d6-b72c0a4b744d: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: e7d3444b-a548-46f0-b8d6-b72c0a4b744d: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758296303, 9), 'signature': {'hash': b'S\x11\x91\x04gq\x92\xd9\x0cp\x90H$\x01\xd1\xb1:\xcd"]', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758296303, 9)}
2025-09-19 16:38:23,517 - app - INFO - Database connection established
2025-09-19 16:38:23,518 - app - INFO - Services initialized
2025-09-19 16:38:35,191 - utils.database - ERROR - Error initializing database indexes: Index build failed: 55d55f82-e2a7-46d1-9171-c7045fc37d01: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: 55d55f82-e2a7-46d1-9171-c7045fc37d01: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758296315, 5), 'signature': {'hash': b'$\xa7\xc0\xb5\xde\xd7\xba\xa6\xc3\xad}\x04h\x91\x9c\x1d\xd2\x85\x7f\xf2', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758296315, 5)}
2025-09-19 16:38:35,193 - app - INFO - Database connection established
2025-09-19 16:38:35,194 - app - INFO - Services initialized
2025-09-19 16:38:35,511 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:40:42,377 - utils.database - ERROR - Error initializing database indexes: Index build failed: cae3250b-c40d-4709-a578-bced64cb6a50: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: cae3250b-c40d-4709-a578-bced64cb6a50: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758296442, 11), 'signature': {'hash': b'\x1e\xe9\xf3%\xd2\xd7V\x87\xf5\xb0\xd8;:\xf01.%d\xc7W', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758296442, 11)}
2025-09-19 16:40:42,379 - app - INFO - Database connection established
2025-09-19 16:40:42,385 - app - INFO - Services initialized
2025-09-19 16:40:42,836 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:50:26,187 - utils.database - ERROR - Error initializing database indexes: Index build failed: 1ff4e8c5-6f24-48df-a680-e2da8cf3d8fe: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: 1ff4e8c5-6f24-48df-a680-e2da8cf3d8fe: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758297026, 7), 'signature': {'hash': b'\xa5\xe9\x10R\xfb\xa0\xde\x99i^\xb2C\xa5\xfbH\xf8\xbc\x93\x93\x07', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297026, 7)}
2025-09-19 16:50:26,193 - app - INFO - Database connection established
2025-09-19 16:50:26,197 - app - INFO - Services initialized
2025-09-19 16:50:35,298 - utils.database - ERROR - Error initializing database indexes: Index build failed: 1bc2b742-ed07-4b8e-abcf-1488ee8c8774: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: 1bc2b742-ed07-4b8e-abcf-1488ee8c8774: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758297035, 9), 'signature': {'hash': b'\x81\x1d\xb6res_\xe1\xfc\xde\\\x94\xc8\xd4\x08^$\x1f\xd3W', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297035, 9)}
2025-09-19 16:50:35,300 - app - INFO - Database connection established
2025-09-19 16:50:35,301 - app - INFO - Services initialized
2025-09-19 16:50:35,599 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:59:15,614 - utils.database - ERROR - Error initializing database indexes: Index build failed: 643a8907-bbe7-4a58-912f-4a1cfee5fa51: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: 643a8907-bbe7-4a58-912f-4a1cfee5fa51: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758297555, 9), 'signature': {'hash': b'EL(AP\xc1\xc1z\xe2c\x02\xfb\xc6\x13\x96\xca8\n\xa7\xab', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297555, 9)}
2025-09-19 16:59:15,616 - app - INFO - Database connection established
2025-09-19 16:59:15,616 - app - INFO - Services initialized
2025-09-19 16:59:15,909 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 16:59:44,876 - utils.database - ERROR - Error initializing database indexes: Index build failed: 36aa3eab-e864-437d-8b30-b349283720ab: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: 36aa3eab-e864-437d-8b30-b349283720ab: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758297584, 20), 'signature': {'hash': b'D,\xc4\x9c\x9b8*=\xde\xb7\x0b\xa5\xc7(\xa8lu\xfe\xc3\x12', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297584, 20)}
2025-09-19 16:59:44,891 - app - INFO - Database connection established
2025-09-19 16:59:44,892 - app - INFO - Services initialized
2025-09-19 16:59:45,701 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 17:00:03,254 - utils.database - ERROR - Error initializing database indexes: Index build failed: 835d0c8a-670a-4741-a0d3-a4419071e6c8: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: 835d0c8a-670a-4741-a0d3-a4419071e6c8: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758297603, 10), 'signature': {'hash': b'?\xb2\x16\r\xc0G\x88\x91\x91\xe9\xe8\x81A\x80\xf04\xb2i\xc4\xa8', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297603, 10)}
2025-09-19 17:00:03,257 - app - INFO - Database connection established
2025-09-19 17:00:03,259 - app - INFO - Services initialized
2025-09-19 17:00:03,606 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 17:00:26,108 - utils.database - ERROR - Error initializing database indexes: Index build failed: dd756dd5-06b3-4513-80ae-8c67cfca406d: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: dd756dd5-06b3-4513-80ae-8c67cfca406d: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758297625, 25), 'signature': {'hash': b'\x9e{\xe1\xb9W\xba9\x91\x94i\x9a4\x84\xf2I9\xf4\xf4/\xb5', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297625, 25)}
2025-09-19 17:00:26,110 - app - INFO - Database connection established
2025-09-19 17:00:26,111 - app - INFO - Services initialized
2025-09-19 17:03:50,871 - utils.database - ERROR - Error initializing database indexes: Index build failed: 4ba2b45c-a47a-45bd-b4ef-2de19c85a9fa: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: 4ba2b45c-a47a-45bd-b4ef-2de19c85a9fa: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: serial_number_1 dup key: { serial_number: "ABCV-25-0607" }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'serial_number': 1}, 'keyValue': {'serial_number': 'ABCV-25-0607'}, '$clusterTime': {'clusterTime': Timestamp(1758297830, 18), 'signature': {'hash': b'^\x8bJl\xd8\xa81\xc0X\x1bM\x85\x9e\xf7\x07\x93\xa8gl\xc8', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297830, 18)}
2025-09-19 17:03:50,873 - app - INFO - Database connection established
2025-09-19 17:03:50,877 - app - INFO - Services initialized
2025-09-19 17:04:02,287 - __main__ - INFO - Database indexes initialized successfully
2025-09-19 17:04:12,530 - utils.database - ERROR - Error initializing database indexes: Index build failed: abc9e12c-0bec-4b7a-852f-fce63945f995: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: public_key_1 dup key: { public_key: null }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: abc9e12c-0bec-4b7a-852f-fce63945f995: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: public_key_1 dup key: { public_key: null }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'public_key': 1}, 'keyValue': {'public_key': None}, '$clusterTime': {'clusterTime': Timestamp(1758297852, 30), 'signature': {'hash': b'\xdc/\xf4%\x1b\xbd\xd3\xff\x1cEv\x16\x9e\xca\xb0&\xecJ\x88,', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297852, 30)}
2025-09-19 17:04:12,533 - app - INFO - Database connection established
2025-09-19 17:04:12,534 - app - INFO - Services initialized
2025-09-19 17:04:17,679 - __main__ - INFO - Database indexes initialized successfully
2025-09-19 17:04:17,738 - werkzeug - WARNING -  * Debugger is active!
2025-09-19 17:04:45,821 - utils.database - ERROR - Error initializing database indexes: Index build failed: 6fa5c0e6-53f2-4c93-a887-01029659eba2: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: public_key_1 dup key: { public_key: null }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: 6fa5c0e6-53f2-4c93-a887-01029659eba2: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: public_key_1 dup key: { public_key: null }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'public_key': 1}, 'keyValue': {'public_key': None}, '$clusterTime': {'clusterTime': Timestamp(1758297885, 41), 'signature': {'hash': b',\xc2\xd4[8\xf6\xa7y+\x87I(A\xc4\xf6\xdcA\xbf7k', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297885, 41)}
2025-09-19 17:04:45,822 - app - INFO - Database connection established
2025-09-19 17:04:45,823 - app - INFO - Services initialized
2025-09-19 17:05:29,862 - __main__ - INFO - Database indexes initialized successfully
2025-09-19 17:05:57,263 - utils.database - ERROR - Error initializing database indexes: Index build failed: a3bfd8d4-a862-4bdb-8d7b-bc17c5bc5e80: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: public_key_1 dup key: { public_key: null }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: a3bfd8d4-a862-4bdb-8d7b-bc17c5bc5e80: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: public_key_1 dup key: { public_key: null }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'public_key': 1}, 'keyValue': {'public_key': None}, '$clusterTime': {'clusterTime': Timestamp(1758297957, 14), 'signature': {'hash': b'\x87\xe3r\xb3\x103l\t5x#\x9f\xe6\x8fK\x80O\x0bo\x81', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297957, 14)}
2025-09-19 17:05:57,264 - app - INFO - Database connection established
2025-09-19 17:05:57,265 - app - INFO - Services initialized
2025-09-19 17:06:13,525 - utils.database - ERROR - Error initializing database indexes: Index build failed: 827c9db3-50a3-42d4-8e65-2340fe096334: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: public_key_1 dup key: { public_key: null }, full error: {'ok': 0.0, 'errmsg': 'Index build failed: 827c9db3-50a3-42d4-8e65-2340fe096334: Collection product_verification.products ( 7c3844be-5f05-41ce-8106-a6f93eaaa511 ) :: caused by :: E11000 duplicate key error collection: product_verification.products index: public_key_1 dup key: { public_key: null }', 'code': 11000, 'codeName': 'DuplicateKey', 'keyPattern': {'public_key': 1}, 'keyValue': {'public_key': None}, '$clusterTime': {'clusterTime': Timestamp(1758297973, 20), 'signature': {'hash': b'v\xef\x19\xf3\\\x8dG\xd5*\x95\xadl\x13\x1fY\np\x86p\x94', 'keyId': 7501759904063946760}}, 'operationTime': Timestamp(1758297973, 20)}
2025-09-19 17:06:13,532 - app - INFO - Database connection established
2025-09-19 17:06:13,535 - app - INFO - Services initialized
2025-09-19 17:06:14,079 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 02:24:13,218 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 02:24:13,218 - app - INFO - Database connection established
2025-09-20 02:24:13,219 - app - INFO - Services initialized
2025-09-20 02:24:27,216 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 02:24:27,217 - app - INFO - Database connection established
2025-09-20 02:24:27,218 - app - INFO - Services initialized
2025-09-20 02:24:27,566 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 02:50:48,996 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 02:50:48,997 - app - INFO - Database connection established
2025-09-20 02:50:48,997 - app - INFO - Services initialized
2025-09-20 02:50:49,367 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 02:56:50,112 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 02:56:50,113 - app - INFO - Database connection established
2025-09-20 02:56:50,114 - app - INFO - Services initialized
2025-09-20 02:56:50,553 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 02:58:35,026 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 02:58:35,027 - app - INFO - Database connection established
2025-09-20 02:58:35,028 - app - INFO - Services initialized
2025-09-20 02:58:35,223 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 02:59:02,083 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 02:59:02,084 - app - INFO - Database connection established
2025-09-20 02:59:02,085 - app - INFO - Services initialized
2025-09-20 03:12:57,490 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 03:12:57,491 - app - INFO - Database connection established
2025-09-20 03:12:57,491 - app - INFO - Services initialized
2025-09-20 03:12:57,625 - services.crypto_service - INFO - Generated new master encryption key
2025-09-20 03:13:09,772 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 03:13:09,773 - app - INFO - Database connection established
2025-09-20 03:13:09,774 - app - INFO - Services initialized
2025-09-20 03:13:10,063 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 03:18:26,505 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 03:18:26,510 - app - INFO - Database connection established
2025-09-20 03:18:26,512 - app - INFO - Services initialized
2025-09-20 03:18:27,355 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 03:18:59,155 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 03:18:59,163 - app - INFO - Database connection established
2025-09-20 03:18:59,163 - app - INFO - Services initialized
2025-09-20 03:18:59,912 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 03:19:27,947 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 03:19:27,955 - app - INFO - Database connection established
2025-09-20 03:19:27,966 - app - INFO - Services initialized
2025-09-20 03:19:28,887 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 03:20:09,006 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 03:20:09,036 - app - INFO - Database connection established
2025-09-20 03:20:09,037 - app - INFO - Services initialized
2025-09-20 03:20:10,085 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:23:52,376 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:23:52,418 - app - INFO - Database connection established
2025-09-20 04:23:52,420 - app - INFO - Services initialized
2025-09-20 04:24:38,387 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:24:38,389 - app - INFO - Database connection established
2025-09-20 04:24:38,395 - app - INFO - Services initialized
2025-09-20 04:24:39,463 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:29:48,768 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:29:48,769 - app - INFO - Database connection established
2025-09-20 04:29:48,770 - app - INFO - Services initialized
2025-09-20 04:30:00,416 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:30:00,419 - app - INFO - Database connection established
2025-09-20 04:30:00,420 - app - INFO - Services initialized
2025-09-20 04:30:00,738 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:38:43,469 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:38:43,470 - app - INFO - Database connection established
2025-09-20 04:38:43,471 - app - INFO - Services initialized
2025-09-20 04:38:44,129 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:39:09,031 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:39:09,033 - app - INFO - Database connection established
2025-09-20 04:39:09,033 - app - INFO - Services initialized
2025-09-20 04:39:09,280 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:39:22,393 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:39:22,394 - app - INFO - Database connection established
2025-09-20 04:39:22,395 - app - INFO - Services initialized
2025-09-20 04:39:34,623 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:39:34,624 - app - INFO - Database connection established
2025-09-20 04:39:34,624 - app - INFO - Services initialized
2025-09-20 04:39:35,185 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:41:51,734 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:41:51,736 - app - INFO - Database connection established
2025-09-20 04:41:51,736 - app - INFO - Services initialized
2025-09-20 04:41:52,137 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:44:29,048 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:44:29,049 - app - INFO - Database connection established
2025-09-20 04:44:29,050 - app - INFO - Services initialized
2025-09-20 04:44:40,556 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:44:40,558 - app - INFO - Database connection established
2025-09-20 04:44:40,558 - app - INFO - Services initialized
2025-09-20 04:44:40,928 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:47:54,140 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:47:54,143 - app - INFO - Database connection established
2025-09-20 04:47:54,144 - app - INFO - Services initialized
2025-09-20 04:47:54,543 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:49:15,110 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340155110", "method": "GET", "path": "/", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T03:49:15.110138+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Upgrade-Insecure-Requests": "1", "Sec-Fetch-Dest": "document", "Sec-Fetch-Mode": "navigate", "Sec-Fetch-Site": "none", "Sec-Fetch-User": "?1", "Priority": "u=0, i"}}
2025-09-20 04:49:15,242 - middleware.auth_middleware - ERROR - API Error: Resource not found - GET /
2025-09-20 04:49:15,257 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340155110", "status_code": 404, "response_time_ms": 146.9, "content_length": 142, "timestamp": "2025-09-20T03:49:15.257013+00:00", "error_details": {"error": "Resource not found", "method": "GET", "path": "/", "status": "error", "timestamp": "2025-09-20T03:49:15.241828+00:00"}}
2025-09-20 04:49:15,863 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340155863", "method": "GET", "path": "/favicon.ico", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T03:49:15.863175+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "image/avif,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Referer": "http://localhost:5000/", "Sec-Fetch-Dest": "image", "Sec-Fetch-Mode": "no-cors", "Sec-Fetch-Site": "cross-site", "Priority": "u=6"}}
2025-09-20 04:49:15,994 - middleware.auth_middleware - ERROR - API Error: Resource not found - GET /favicon.ico
2025-09-20 04:49:15,998 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340155863", "status_code": 404, "response_time_ms": 134.64, "content_length": 153, "timestamp": "2025-09-20T03:49:15.997820+00:00", "error_details": {"error": "Resource not found", "method": "GET", "path": "/favicon.ico", "status": "error", "timestamp": "2025-09-20T03:49:15.993834+00:00"}}
2025-09-20 04:51:55,250 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:51:55,259 - app - INFO - Database connection established
2025-09-20 04:51:55,260 - app - INFO - Services initialized
2025-09-20 04:51:55,978 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:52:28,758 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:52:28,766 - app - INFO - Database connection established
2025-09-20 04:52:28,770 - app - INFO - Services initialized
2025-09-20 04:52:45,506 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:52:45,513 - app - INFO - Database connection established
2025-09-20 04:52:45,513 - app - INFO - Services initialized
2025-09-20 04:52:45,862 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:52:57,614 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340377614", "method": "GET", "path": "/", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T03:52:57.614155+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Upgrade-Insecure-Requests": "1", "Sec-Fetch-Dest": "document", "Sec-Fetch-Mode": "navigate", "Sec-Fetch-Site": "none", "Sec-Fetch-User": "?1", "Priority": "u=0, i"}}
2025-09-20 04:52:57,728 - middleware.auth_middleware - ERROR - API Error: Resource not found - GET /
2025-09-20 04:52:57,743 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340377614", "status_code": 404, "response_time_ms": 128.85, "content_length": 142, "timestamp": "2025-09-20T03:52:57.742986+00:00", "error_details": {"error": "Resource not found", "method": "GET", "path": "/", "status": "error", "timestamp": "2025-09-20T03:52:57.728045+00:00"}}
2025-09-20 04:52:58,094 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340378093", "method": "GET", "path": "/favicon.ico", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T03:52:58.093993+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "image/avif,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Referer": "http://localhost:5000/", "Sec-Fetch-Dest": "image", "Sec-Fetch-Mode": "no-cors", "Sec-Fetch-Site": "cross-site", "Priority": "u=6"}}
2025-09-20 04:52:58,211 - middleware.auth_middleware - ERROR - API Error: Resource not found - GET /favicon.ico
2025-09-20 04:52:58,214 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340378093", "status_code": 404, "response_time_ms": 119.6, "content_length": 153, "timestamp": "2025-09-20T03:52:58.213580+00:00", "error_details": {"error": "Resource not found", "method": "GET", "path": "/favicon.ico", "status": "error", "timestamp": "2025-09-20T03:52:58.211048+00:00"}}
2025-09-20 04:54:26,696 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:54:26,699 - app - INFO - Database connection established
2025-09-20 04:54:26,700 - app - INFO - Services initialized
2025-09-20 04:54:41,469 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:54:41,471 - app - INFO - Database connection established
2025-09-20 04:54:41,471 - app - INFO - Services initialized
2025-09-20 04:54:41,775 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:55:08,962 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340508961", "method": "GET", "path": "/", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T03:55:08.961809+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Upgrade-Insecure-Requests": "1", "Sec-Fetch-Dest": "document", "Sec-Fetch-Mode": "navigate", "Sec-Fetch-Site": "none", "Sec-Fetch-User": "?1", "Priority": "u=0, i"}}
2025-09-20 04:55:09,146 - middleware.auth_middleware - ERROR - API Error: Resource not found - GET /
2025-09-20 04:55:09,149 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340508961", "status_code": 404, "response_time_ms": 187.23, "content_length": 142, "timestamp": "2025-09-20T03:55:09.148994+00:00", "error_details": {"error": "Resource not found", "method": "GET", "path": "/", "status": "error", "timestamp": "2025-09-20T03:55:09.145977+00:00"}}
2025-09-20 04:55:09,650 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340509649", "method": "GET", "path": "/favicon.ico", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T03:55:09.649664+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "image/avif,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Referer": "http://localhost:5000/", "Sec-Fetch-Dest": "image", "Sec-Fetch-Mode": "no-cors", "Sec-Fetch-Site": "cross-site", "Priority": "u=6"}}
2025-09-20 04:55:09,835 - middleware.auth_middleware - ERROR - API Error: Resource not found - GET /favicon.ico
2025-09-20 04:55:09,837 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340509649", "status_code": 404, "response_time_ms": 187.22, "content_length": 153, "timestamp": "2025-09-20T03:55:09.836835+00:00", "error_details": {"error": "Resource not found", "method": "GET", "path": "/favicon.ico", "status": "error", "timestamp": "2025-09-20T03:55:09.835419+00:00"}}
2025-09-20 04:57:03,273 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:57:03,278 - app - INFO - Database connection established
2025-09-20 04:57:03,285 - app - INFO - Services initialized
2025-09-20 04:57:19,506 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 04:57:19,515 - app - INFO - Database connection established
2025-09-20 04:57:19,520 - app - INFO - Services initialized
2025-09-20 04:57:20,002 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 04:57:20,145 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340640145", "method": "GET", "path": "/", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T03:57:20.145468+00:00", "query_params": {}, "headers": {"Host": "127.0.0.1:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Upgrade-Insecure-Requests": "1", "Sec-Fetch-Dest": "document", "Sec-Fetch-Mode": "navigate", "Sec-Fetch-Site": "none", "Sec-Fetch-User": "?1", "Priority": "u=0, i"}}
2025-09-20 04:57:20,341 - middleware.auth_middleware - ERROR - API Error: Resource not found - GET /
2025-09-20 04:57:20,345 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340640145", "status_code": 404, "response_time_ms": 198.73, "content_length": 142, "timestamp": "2025-09-20T03:57:20.344245+00:00", "error_details": {"error": "Resource not found", "method": "GET", "path": "/", "status": "error", "timestamp": "2025-09-20T03:57:20.341498+00:00"}}
2025-09-20 04:57:20,888 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340640887", "method": "GET", "path": "/favicon.ico", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T03:57:20.887677+00:00", "query_params": {}, "headers": {"Host": "127.0.0.1:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "image/avif,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Referer": "http://127.0.0.1:5000/", "Sec-Fetch-Dest": "image", "Sec-Fetch-Mode": "no-cors", "Sec-Fetch-Site": "cross-site", "Priority": "u=6"}}
2025-09-20 04:57:21,080 - middleware.auth_middleware - ERROR - API Error: Resource not found - GET /favicon.ico
2025-09-20 04:57:21,082 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340640887", "status_code": 404, "response_time_ms": 194.62, "content_length": 153, "timestamp": "2025-09-20T03:57:21.082227+00:00", "error_details": {"error": "Resource not found", "method": "GET", "path": "/favicon.ico", "status": "error", "timestamp": "2025-09-20T03:57:21.080625+00:00"}}
2025-09-20 05:02:21,385 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 05:02:21,387 - app - INFO - Database connection established
2025-09-20 05:02:21,387 - app - INFO - Services initialized
2025-09-20 05:02:21,742 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 05:02:49,570 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 05:02:49,572 - app - INFO - Database connection established
2025-09-20 05:02:49,573 - app - INFO - Services initialized
2025-09-20 05:02:50,077 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 05:02:50,329 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340970328", "method": "GET", "path": "/", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T04:02:50.328891+00:00", "query_params": {}, "headers": {"Host": "127.0.0.1:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Upgrade-Insecure-Requests": "1", "Sec-Fetch-Dest": "document", "Sec-Fetch-Mode": "navigate", "Sec-Fetch-Site": "none", "Sec-Fetch-User": "?1", "Priority": "u=0, i"}}
2025-09-20 05:02:50,467 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340970328", "status_code": 200, "response_time_ms": 135.33, "content_length": 367, "timestamp": "2025-09-20T04:02:50.464214+00:00"}
2025-09-20 05:02:50,834 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758340970831", "method": "GET", "path": "/favicon.ico", "client_ip": "127.0.0.1", "user_agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "timestamp": "2025-09-20T04:02:50.831807+00:00", "query_params": {}, "headers": {"Host": "127.0.0.1:5000", "User-Agent": "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:142.0) Gecko/20100101 Firefox/142.0", "Accept": "image/avif,image/webp,image/png,image/svg+xml,image/*;q=0.8,*/*;q=0.5", "Accept-Language": "en-GB,en;q=0.5", "Accept-Encoding": "gzip, deflate, br, zstd", "Connection": "keep-alive", "Referer": "http://127.0.0.1:5000/", "Sec-Fetch-Dest": "image", "Sec-Fetch-Mode": "no-cors", "Sec-Fetch-Site": "cross-site", "Priority": "u=6"}}
2025-09-20 05:02:50,946 - middleware.auth_middleware - ERROR - API Error: Resource not found - GET /favicon.ico
2025-09-20 05:02:50,949 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758340970831", "status_code": 404, "response_time_ms": 116.98, "content_length": 153, "timestamp": "2025-09-20T04:02:50.948724+00:00", "error_details": {"error": "Resource not found", "method": "GET", "path": "/favicon.ico", "status": "error", "timestamp": "2025-09-20T04:02:50.945813+00:00"}}
2025-09-20 05:03:37,202 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 05:03:37,203 - app - INFO - Database connection established
2025-09-20 05:03:37,204 - app - INFO - Services initialized
2025-09-20 05:04:20,845 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 05:04:20,847 - app - INFO - Database connection established
2025-09-20 05:04:20,847 - app - INFO - Services initialized
2025-09-20 05:04:32,248 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 05:04:32,249 - app - INFO - Database connection established
2025-09-20 05:04:32,250 - app - INFO - Services initialized
2025-09-20 05:04:32,599 - werkzeug - WARNING -  * Debugger is active!
2025-09-20 06:34:31,221 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 06:34:31,222 - app - INFO - Database connection established
2025-09-20 06:34:31,224 - app - INFO - Services initialized
2025-09-20 06:34:50,922 - utils.database - INFO - Database indexes initialized successfully
2025-09-20 06:34:50,927 - app - INFO - Database connection established
2025-09-20 06:34:50,928 - app - INFO - Services initialized
2025-09-20 06:34:51,382 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 20:41:38,278 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 20:41:38,280 - app - INFO - Database connection established
2025-09-24 20:41:38,282 - app - INFO - Services initialized
2025-09-24 20:42:02,654 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 20:42:02,655 - app - INFO - Database connection established
2025-09-24 20:42:02,655 - app - INFO - Services initialized
2025-09-24 20:42:03,801 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 20:52:15,780 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 20:52:15,781 - app - INFO - Database connection established
2025-09-24 20:52:15,782 - app - INFO - Services initialized
2025-09-24 20:52:29,793 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 20:52:29,795 - app - INFO - Database connection established
2025-09-24 20:52:29,796 - app - INFO - Services initialized
2025-09-24 20:52:30,794 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 21:44:19,947 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 21:44:19,948 - app - INFO - Database connection established
2025-09-24 21:44:19,949 - app - INFO - Services initialized
2025-09-24 21:44:20,709 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 21:45:00,205 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 21:45:00,209 - app - INFO - Database connection established
2025-09-24 21:45:00,211 - app - INFO - Services initialized
2025-09-24 21:45:00,903 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 21:45:22,329 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 21:45:22,331 - app - INFO - Database connection established
2025-09-24 21:45:22,331 - app - INFO - Services initialized
2025-09-24 21:45:23,048 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 21:45:46,557 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 21:45:46,559 - app - INFO - Database connection established
2025-09-24 21:45:46,560 - app - INFO - Services initialized
2025-09-24 21:45:47,596 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 21:56:09,671 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 21:56:09,673 - app - INFO - Database connection established
2025-09-24 21:56:09,673 - app - INFO - Services initialized
2025-09-24 21:56:21,085 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 21:56:21,086 - app - INFO - Database connection established
2025-09-24 21:56:21,087 - app - INFO - Services initialized
2025-09-24 21:56:21,706 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 21:58:12,413 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 21:58:12,414 - app - INFO - Database connection established
2025-09-24 21:58:12,415 - app - INFO - Services initialized
2025-09-24 21:58:28,583 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 21:58:28,584 - app - INFO - Database connection established
2025-09-24 21:58:28,585 - app - INFO - Services initialized
2025-09-24 21:58:29,236 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 21:59:08,668 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 21:59:08,670 - app - INFO - Database connection established
2025-09-24 21:59:08,671 - app - INFO - Services initialized
2025-09-24 21:59:09,231 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:03:55,808 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:03:55,809 - app - INFO - Database connection established
2025-09-24 22:03:55,809 - app - INFO - Services initialized
2025-09-24 22:03:56,368 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:10:40,720 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:10:40,721 - app - INFO - Database connection established
2025-09-24 22:10:40,722 - app - INFO - Services initialized
2025-09-24 22:10:41,338 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:13:04,296 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:13:04,304 - app - INFO - Database connection established
2025-09-24 22:13:04,310 - app - INFO - Services initialized
2025-09-24 22:13:07,456 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:14:56,550 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:14:56,552 - app - INFO - Database connection established
2025-09-24 22:14:56,553 - app - INFO - Services initialized
2025-09-24 22:14:57,184 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:15:46,133 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:15:46,134 - app - INFO - Database connection established
2025-09-24 22:15:46,135 - app - INFO - Services initialized
2025-09-24 22:15:46,714 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:16:18,193 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:16:18,194 - app - INFO - Database connection established
2025-09-24 22:16:18,194 - app - INFO - Services initialized
2025-09-24 22:16:18,977 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:23:37,425 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758749017425", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T21:23:37.425290+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*"}}
2025-09-24 22:23:37,612 - middleware.auth_middleware - ERROR - API Error: Resource not found - POST /auth/admin/login
2025-09-24 22:23:37,616 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758749017425", "status_code": 404, "response_time_ms": 190.7, "content_length": 159, "timestamp": "2025-09-24T21:23:37.615981+00:00", "error_details": {"error": "Resource not found", "method": "POST", "path": "/auth/admin/login", "status": "error", "timestamp": "2025-09-24T21:23:37.611894+00:00"}}
2025-09-24 22:24:43,271 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:24:43,271 - app - INFO - Database connection established
2025-09-24 22:24:43,272 - app - INFO - Services initialized
2025-09-24 22:24:44,035 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:24:45,156 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758749085155", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T21:24:45.155796+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*"}}
2025-09-24 22:24:45,373 - middleware.auth_middleware - ERROR - API Error: Resource not found - POST /auth/admin/login
2025-09-24 22:24:45,380 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758749085155", "status_code": 404, "response_time_ms": 224.43, "content_length": 159, "timestamp": "2025-09-24T21:24:45.380192+00:00", "error_details": {"error": "Resource not found", "method": "POST", "path": "/auth/admin/login", "status": "error", "timestamp": "2025-09-24T21:24:45.373412+00:00"}}
2025-09-24 22:25:14,035 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:25:14,036 - app - INFO - Database connection established
2025-09-24 22:25:14,038 - app - INFO - Services initialized
2025-09-24 22:25:14,746 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:26:47,235 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:26:47,237 - app - INFO - Database connection established
2025-09-24 22:26:47,238 - app - INFO - Services initialized
2025-09-24 22:27:04,702 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:27:04,704 - app - INFO - Database connection established
2025-09-24 22:27:04,708 - app - INFO - Services initialized
2025-09-24 22:27:05,933 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:30:32,921 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:30:32,922 - app - INFO - Database connection established
2025-09-24 22:30:32,922 - app - INFO - Services initialized
2025-09-24 22:30:33,953 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:30:54,429 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:30:54,431 - app - INFO - Database connection established
2025-09-24 22:30:54,432 - app - INFO - Services initialized
2025-09-24 22:39:51,817 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:39:51,818 - app - INFO - Database connection established
2025-09-24 22:39:51,819 - app - INFO - Services initialized
2025-09-24 22:40:12,179 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:40:12,181 - app - INFO - Database connection established
2025-09-24 22:40:12,181 - app - INFO - Services initialized
2025-09-24 22:40:12,783 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:40:19,681 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758750019680", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T21:40:19.681088+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*"}}
2025-09-24 22:40:19,881 - routes.auth_routes - ERROR - Admin login error: 415 Unsupported Media Type: Did not attempt to load JSON data because the request Content-Type was not 'application/json'.
2025-09-24 22:40:19,884 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758750019680", "status_code": 500, "response_time_ms": 202.78, "content_length": 30, "timestamp": "2025-09-24T21:40:19.883845+00:00", "error_details": {"error": "Login failed"}}
2025-09-24 22:48:46,189 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:48:46,192 - app - INFO - Database connection established
2025-09-24 22:48:46,195 - app - INFO - Services initialized
2025-09-24 22:48:47,361 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:48:47,709 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758750527708", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T21:48:47.708565+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*"}}
2025-09-24 22:48:47,906 - routes.auth_routes - ERROR - Admin login error: 415 Unsupported Media Type: Did not attempt to load JSON data because the request Content-Type was not 'application/json'.
2025-09-24 22:48:47,912 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758750527708", "status_code": 500, "response_time_ms": 202.23, "content_length": 30, "timestamp": "2025-09-24T21:48:47.910764+00:00", "error_details": {"error": "Login failed"}}
2025-09-24 22:50:38,272 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:50:38,273 - app - INFO - Database connection established
2025-09-24 22:50:38,273 - app - INFO - Services initialized
2025-09-24 22:50:39,026 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:51:36,870 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:51:36,871 - app - INFO - Database connection established
2025-09-24 22:51:36,872 - app - INFO - Services initialized
2025-09-24 22:51:37,776 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 22:53:02,341 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758750782339", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T21:53:02.339927+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*", "Content-Type": "application/json", "Content-Length": "66"}, "body": {"email": "damilolaoniyide11@gmail.com"}}
2025-09-24 22:53:02,702 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758750782339", "status_code": 401, "response_time_ms": 362.52, "content_length": 37, "timestamp": "2025-09-24T21:53:02.702342+00:00", "error_details": {"error": "Invalid credentials"}}
2025-09-24 22:56:43,473 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 22:56:43,474 - app - INFO - Database connection established
2025-09-24 22:56:43,475 - app - INFO - Services initialized
2025-09-24 22:56:44,392 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 23:01:32,445 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758751292443", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T22:01:32.443977+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*", "Content-Type": "application/json", "Content-Length": "66"}, "body": {"email": "damilolaoniyide11@gmail.com"}}
2025-09-24 23:01:32,825 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758751292443", "status_code": 401, "response_time_ms": 381.21, "content_length": 37, "timestamp": "2025-09-24T22:01:32.825285+00:00", "error_details": {"error": "Invalid credentials"}}
2025-09-24 23:05:39,139 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:05:39,140 - app - INFO - Database connection established
2025-09-24 23:05:39,141 - app - INFO - Services initialized
2025-09-24 23:05:39,890 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 23:14:16,185 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:14:16,223 - app - INFO - Database connection established
2025-09-24 23:14:16,223 - app - INFO - Services initialized
2025-09-24 23:14:17,437 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 23:15:38,522 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:15:38,523 - app - INFO - Database connection established
2025-09-24 23:15:38,524 - app - INFO - Services initialized
2025-09-24 23:15:39,356 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 23:19:44,209 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:19:44,220 - app - INFO - Database connection established
2025-09-24 23:19:44,222 - app - INFO - Services initialized
2025-09-24 23:19:45,459 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 23:26:06,085 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:26:06,086 - app - INFO - Database connection established
2025-09-24 23:26:06,087 - app - INFO - Services initialized
2025-09-24 23:26:07,251 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 23:26:27,126 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758752787124", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T22:26:27.125078+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*", "Content-Type": "application/json", "Content-Length": "66"}, "body": {"email": "damilolaoniyide11@gmail.com"}}
2025-09-24 23:26:28,108 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758752787124", "status_code": 401, "response_time_ms": 983.47, "content_length": 37, "timestamp": "2025-09-24T22:26:28.108494+00:00", "error_details": {"error": "Invalid credentials"}}
2025-09-24 23:29:30,891 - app - ERROR - Database initialization failed: cannot access local variable 'get_db_connection' where it is not associated with a value
2025-09-24 23:31:28,402 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:31:28,405 - app - INFO - Database connection established
2025-09-24 23:31:28,406 - app - INFO - Services initialized
2025-09-24 23:34:55,306 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:34:55,307 - app - INFO - Database connection established
2025-09-24 23:34:55,307 - app - INFO - Services initialized
2025-09-24 23:36:37,389 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:36:37,391 - app - INFO - Database connection established
2025-09-24 23:36:37,391 - app - INFO - Services initialized
2025-09-24 23:36:49,251 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:36:49,253 - app - INFO - Database connection established
2025-09-24 23:36:49,253 - app - INFO - Services initialized
2025-09-24 23:36:49,974 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 23:37:10,123 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758753430121", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T22:37:10.122055+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*", "Content-Type": "application/json", "Content-Length": "66"}, "body": {"email": "damilolaoniyide11@gmail.com"}}
2025-09-24 23:37:12,757 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758753430121", "status_code": 401, "response_time_ms": 2635.1, "content_length": 37, "timestamp": "2025-09-24T22:37:12.757104+00:00", "error_details": {"error": "Invalid credentials"}}
2025-09-24 23:39:55,061 - utils.database - INFO - Database indexes initialized successfully
2025-09-24 23:39:55,064 - app - INFO - Database connection established
2025-09-24 23:39:55,069 - app - INFO - Services initialized
2025-09-24 23:39:56,347 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 23:46:54,800 - utils.database - ERROR - Error initializing database indexes: ac-ni4ekos-shard-00-02.aoxl38d.mongodb.net:27017: [Errno 110] Connection timed out
2025-09-24 23:46:54,801 - app - INFO - Database connection established
2025-09-24 23:46:54,803 - app - INFO - Services initialized
2025-09-24 23:46:55,597 - werkzeug - WARNING -  * Debugger is active!
2025-09-24 23:46:55,723 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758754015719", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T22:46:55.719883+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*", "Content-Type": "application/json", "Content-Length": "66"}, "body": {"email": "damilolaoniyide11@gmail.com"}}
2025-09-24 23:46:55,730 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758754015729", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T22:46:55.729473+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*", "Content-Type": "application/json", "Content-Length": "66"}, "body": {"email": "damilolaoniyide11@gmail.com"}}
2025-09-24 23:46:55,741 - middleware.logging_middleware - INFO - REQUEST: {"request_id": "req_1758754015740", "method": "POST", "path": "/auth/admin/login", "client_ip": "127.0.0.1", "user_agent": "curl/8.12.1", "timestamp": "2025-09-24T22:46:55.740623+00:00", "query_params": {}, "headers": {"Host": "localhost:5000", "User-Agent": "curl/8.12.1", "Accept": "*/*", "Content-Type": "application/json", "Content-Length": "66"}, "body": {"email": "damilolaoniyide11@gmail.com"}}
2025-09-24 23:46:57,472 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758754015719", "status_code": 200, "response_time_ms": 1752.07, "content_length": 475, "timestamp": "2025-09-24T22:46:57.471925+00:00"}
2025-09-24 23:46:57,506 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758754015729", "status_code": 200, "response_time_ms": 1776.81, "content_length": 475, "timestamp": "2025-09-24T22:46:57.506299+00:00"}
2025-09-24 23:46:57,667 - middleware.logging_middleware - INFO - RESPONSE: {"request_id": "req_1758754015740", "status_code": 200, "response_time_ms": 1926.75, "content_length": 475, "timestamp": "2025-09-24T22:46:57.667349+00:00"}
2025-09-25 00:08:32,996 - utils.database - INFO - Database indexes initialized successfully
2025-09-25 00:08:32,998 - app - INFO - Database connection established
2025-09-25 00:08:32,999 - app - INFO - Services initialized
2025-09-25 00:08:33,926 - werkzeug - WARNING -  * Debugger is active!
2025-09-25 00:16:54,961 - utils.database - INFO - Database indexes initialized successfully
2025-09-25 00:16:54,977 - app - INFO - Database connection established
2025-09-25 00:16:54,981 - app - INFO - Services initialized
2025-09-25 00:16:57,267 - werkzeug - WARNING -  * Debugger is active!
